pages/api-reference/connect/funding-sources/list-for-external-party.mdx

```diff
@@ -5,13 +5,13 @@ title: List funding sources for an external party
 guide:
   step: 5
 meta:
-  title: List funding sources for an External Party | Dwolla Connect | Dwolla API Reference
-  description: Endpoint for listing funding sources for external parties.
+  title: "Dwolla API Reference | Funding Sources: List funding sources for an External Party"
+  description: "Endpoint for listing funding sources for external parties."
 ---
 
 # List funding-sources for an External Party
 
-Retrieve a list of funding sources that belong to a <span><ContentTooltip preset="external-party">external party</ContentTooltip></span>. By default, all funding sources are returned unless the removed query string parameter is set to false in the request.
+Retrieve a list of funding sources that belong to an External Party. By default, all funding sources are returned in the list and can be identified as active or removed via the `removed` attribute.
```

Comment:
```suggestion
Retrieve a list of funding sources that belong to an <span><ContentTooltip preset="external-party">External Party</ContentTooltip></span>. By default, all funding sources are returned in the list and can be identified as active or removed via the `removed` attribute.
```

Reply:
Added the `external-party` tooltip. Feel free to disregard this suggestion if it doesn't fit here! :)